package.json

```diff
@@ -7,7 +7,7 @@
     "lint": "eslint ./ --fix",
     "migrate": "sequelize db:migrate:undo:all && sequelize db:migrate && sequelize db:seed:all",
     "pretest": "NODE_ENV=test npm run migrate",
-    "test": "NODE_ENV=test nyc mocha --compilers js:babel-register ./server/tests/ --timeout 5000ms --exit --no-deprecation",
+    "test": "NODE_ENV=test nyc mocha --compilers js:babel-register ./server/tests/* --timeout 5000ms --exit --no-deprecation",
```

Comment:
revert this to the line above. and add the missing test into one the corresponding tests/../index.js

Reply:
Done, thanks